qa/L0_infer/infer_test.py

```diff
@@ -39,8 +39,8 @@
     int(os.environ.get('TEST_SYSTEM_SHARED_MEMORY', 0)))
 TEST_CUDA_SHARED_MEMORY = bool(int(os.environ.get('TEST_CUDA_SHARED_MEMORY',
                                                   0)))
-CPU_ONLY = (os.environ.get('TRITON_SERVER_CPU_ONLY') is not None)
-
+#CPU_ONLY = (os.environ.get('TRITON_SERVER_CPU_ONLY') is not None)
```

Comment:
I don't think we should be changing L0_infer test. We are going to need additional testing for "separate triton images for invidual backends" ticket and that should be sufficient for compose.py (at least for now). For "individual backend images" testing as part of TRITON_NIGHTLY CI runs we should invoke a new build step that uses compose.py to create the individual images. We will then need to build some kind of devel container  for each and run L0_infer appropriately.... but all those changes can be done for the other ticket in a different PR.